src/processor/operator/filter.cpp

```diff
@@ -5,6 +5,10 @@ using namespace kuzu::common;
 namespace kuzu {
 namespace processor {
 
+std::string FilterPrintInfo::toString() const {
+    return expression->toString();
```

Comment:
No need to solve in this PR but we seem to have a little bit problem when printing functions, e.g. a > 1 we print as `GREATER_THAN(a)`. Let's fix this at the end of operator printing.